src/control/cmd/daos_server/storage_nvme_test.go

```diff
@@ -516,7 +511,8 @@ func TestDaosServer_resetNVMe(t *testing.T) {
 				Reset_:       true,
 			}
 
-			gotErr := resetNVMe(req, &tc.resetCmd.nvmeCmd, scs.NvmePrepare)
+			gotErr := resetNVMe(req, &tc.resetCmd.nvmeCmd, !tc.iommuDisabled,
+				scs.NvmePrepare)
```

Comment:
question - Wondering a bit about the motivation for making iommuDisabled a param, rather than using the iommu checker. Is checking the status expensive?

Reply:
Its about mocking, will try to rework.

Reply:
have tried to address your question